### PR TITLE
[TASK] #101053 - Remove deprecated code from ExpressionLanguage

### PR DESCRIPTION
**Pull request by brotkrueml**
Resolves: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/534
Releases: main

### CODE REVIEW

File: Documentation/Conditions/Index.rst [hunk 1]
```diff
@@ -795,73 +795,6 @@ compatVersion()
       [compatVersion("11.5.1")]
 
 
-.. index:: Conditions; loginUser
-.. _condition-function-loginUser:
-
-loginUser()
-===========
-
-..  deprecated:: 12.4
```
Comment (linawolf) [inline Documentation/Conditions/Index.rst:804]: keep a hint or migration information maybe?

Comment (brotkrueml) [reply Documentation/Conditions/Index.rst:804]: I would remove all that old stuff.

Comment (linawolf) [reply Documentation/Conditions/Index.rst:804]: Migrating Conditions has caused a lot of trouble for users in the past

Comment (brotkrueml) [reply Documentation/Conditions/Index.rst:804]: Okay okay, will leave it in somehow

Comment (linawolf) [reply Documentation/Conditions/Index.rst:804]: We could maybe also make a "Migration" section, we had that in the past

Comment (brotkrueml) [reply Documentation/Conditions/Index.rst:804]: It is already in place in v12. Will just adjust the note above and leave all the stuff the same.

**Review by linawolf — APPROVED** [review]

**Pull request merged**